setup.sh

```diff
@@ -194,6 +194,7 @@ for i in $(echo "
     torchvision
     apex
     ort
+    utf8proc
```

Comment:
We should have the same relative order here as in docker.

Reply:
got it. will update it